docs/INSTALL.md

```diff
@@ -11,9 +11,50 @@ Before you begin the installation process, please make sure you have all the bhi
  2. [Redis](redis.io)
  3. [NodeJS](https://nodejs.org/en/) (we recommend using [node version manager](https://github.com/creationix/nvm) on linux.  Note that we only test on stable and edge).
  4. [WKHTMLtoPDF](http://wkhtmltopdf.org/downloads.html) (use the compiled binaries, even if it is distributed with your package manager.  The binaries come with patched Qt).
+ 5.	npm
+ 6.	curl
+ 7.	yarn
+ 8.	git
+ 
+###### Detailed dependency installation instructions for Ubuntu (verified / installed specifically using VirtualBox)
+
+```bash
+#Run the following command to update the package lists:
+sudo apt-get update
+
+#Install MySQL with the following command:
+sudo apt-get install mysql-server
+
+#Run the following commands to install Redis:
+sudo apt-get install redis-server
+
+#Run the following command to install NodeJS:
+sudo apt-get install nodejs
```

Comment:
In point 3, we recommend using [nvm](https://github.com/creationix/nvm), but these instructions do not use it.  My preference is to encourage users to use node version manager, since that guarantees what version of node they get when the install (their distro's package repositories may be terribly out of date).

I propose you update these instructions by moving the nodejs installation instructions below the wkhtmltopdf and use nvm to install node version 8 (the highest version we support without warnings).  Note that the installation uses `curl`, so curl will need to be installed before node.

```sh
#Install node version manager locally
curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash

#Set up the environmental variables for node version manager
export NVM_DIR="$HOME/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm

#Download NodeJS version 8
nvm install 8
```